v0/destinations/facebook_pixel/transform.js

```diff
@@ -527,6 +527,25 @@ const responseBuilderSimple = (message, category, destination) => {
     }
   }
 
+  // content_category should only be a string ref: https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/custom-data
+
+  if (
+    customData &&
+    customData.content_category &&
+    typeof customData.content_category !== "string"
+  ) {
+    if (Array.isArray(customData.content_category)) {
```

Comment:
Why do we want to handle this case?

Reply:
We have this kind of handling for `product.categories` for E-Com spec in other integrations.
People might use same ut for multiple dest where they map E-com spec where categories are sent as Array we should handle that to keep consistency

Reply:
@saikumarrs  Well, there is no other intention other than trying to accommodate most of the data that the user is sending. Have asked to update that in the doc.